### PR TITLE
Jw/documentation

### PR DESCRIPTION
**Pull request by Jakob37**
Work in progress for new documentation. Suggestions are welcome.

Remaining steps:

- [x] Check the DPAF freq and update outputs
- [x] Go through the remaining FIXMEs

# Description

What is changed? What is the result of the changes?

fixes # (issue)

## Type of change

- [x] Patch

# Checklist:

## Patch

~~- [ ] Stub run completes without errors or new warnings~~ (no code or data changes)
- [x] I have updated the CHANGELOG

 
# Instructions for the reviewers

- [x] Check the documentation by navigating to the `nextflow_wgs` repository, selecting the `jw/documentation` branch, and from there navigating the text and the hyperlinks (in the `newdocs` folder)

# Fixes from review

- [x] Change to completely fictionalized sample/group ids (sample-a, sample-1 or something similar). (Both in CSV and PNG)

# Post-merge

- [x] The `master` branch has been tagged with the new version number.

### CODE REVIEW

File: newdocs/img/samplesheet.PNG
Comment (alkc) [inline newdocs/img/samplesheet.PNG:1]: Change to completely fictionalized lab ids (sample-a, sample-1 or something similar).

**Review by alkc — APPROVED** [review]
Good job! It looks great so far!

File: newdocs/img/samplesheet.PNG
Comment (Jakob37) [reply newdocs/img/samplesheet.PNG:1]: OK makes sense. This is just a screenshot of the already present CSV. But I can update in both CSV and screenshot (for my own analyses I use "0000-00" as ID)

**Comment by Jakob37** [comment]
OK, I'll wrap the ID-fixes and merge this as a first iteration.

**Pull request merged**